client/components/dialog/dialog-base.jsx

```diff
@@ -24,13 +29,16 @@ class DialogBase extends Component {
 			);
 
 		return (
-			<Modal isOpen={ this.props.isVisible }
+			<Modal
+				isOpen={ this.props.isVisible }
 				onRequestClose={ this._close }
```

Comment:
Do you think we could get into a race condition here? Perhaps we should preventDefault if we actually remove a component in the esc handler in `CloseOnEscape`